.github/workflows/build.yml

```diff
@@ -275,6 +276,29 @@ jobs:
           tags: "latest,${{ env.RELEASE_VERSION }},${{ env.RELEASE_MAJOR_VERSION }}"
           dockerfile: kubernetes/Dockerfile
           buildoptions: "--no-cache"
+  publish-checkov-admissioncontroller-dockerhub:
+    runs-on: [self-hosted, public, linux, x64]
+    needs: publish-package
+    steps:
+      - uses: actions/checkout@v2
+      - name: update checkov-k8s version
+        run: |
+          checkov_version=${{ needs.publish-package.outputs.version }}
+          checkov_major_version=$(echo "${checkov_version}" | head -c1)
+          echo ::set-env name=RELEASE_VERSION::$(echo $checkov_version)
+          echo ::set-env name=RELEASE_MAJOR_VERSION::$(echo $checkov_major_version)
+          echo $RELEASE_VERSION
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'
```

Comment:
@schosterbarak needed here? You just wrote a check against it 😆

Reply:
it is required for the set-env

Reply:
Aha! If so, let's use the `set-output` [mechanism](https://github.com/bridgecrewio/checkov/blob/e4a42f0fcc9c0301d7337e55cfbaef8a749e0000/.github/workflows/build.yml#L179). Please make sure you add an id to this step, and then reference it where you need. [Example reference](https://github.com/bridgecrewio/checkov/blob/e4a42f0fcc9c0301d7337e55cfbaef8a749e0000/.github/workflows/build.yml#L187)